tests/foreman/cli/test_satellitesync.py

```diff
@@ -1586,6 +1586,60 @@ def test_negative_import_redhat_cv_without_manifest(
             'appropriate subscriptions before importing content.'
         ) in error.value.message
 
+    @pytest.mark.tier2
```

Comment:
```suggestion
    @pytest.mark.tier2
    @pytest.mark.parametrize('setting_update', ['subscription_connection_enabled'], indirect=True)
```

Reply:
@jameerpathan111 All these suggestions look appropriate, but it would break consistency across the whole test suite. I would recommend to leave it as is and leave these suggestions for huge refactoring that I plan in near future.

Reply:
okay